RT-OneEntry/ListTriggerReadThrough.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Microsoft.Azure.Cosmos;
+using Microsoft.Azure.Cosmos.Linq;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Redis;
+using Microsoft.Extensions.Logging;
+using Microsoft.Identity.Client;
+using Newtonsoft.Json.Linq;
+using StackExchange.Redis;
+using System;
+using System.Collections;
+using System.Collections.Concurrent;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.ComponentModel.Design;
+using System.Drawing;
+using System.Linq;
+using System.Reflection.Metadata;
+using System.Text;
+using System.Threading.Tasks;
+using static System.Net.Mime.MediaTypeNames;
+using Container = Microsoft.Azure.Cosmos.Container;
+
+namespace RT_OneEntry
+{
+    public record ListData
+    (
+        string id,
+        List<string> value
+    );
+    public static class ListTriggerReadThrough
+    {
+        public const string localhostSetting = "redisLocalhost";
+        private static readonly IDatabase cache = ConnectionMultiplexer.Connect(Environment.GetEnvironmentVariable(localhostSetting)).GetDatabase();
+
+        [FunctionName(nameof(ListTriggerReadThroughFunc))]
+        public static async Task ListTriggerReadThroughFunc(
+            [RedisPubSubTrigger(localhostSetting, "__keyevent@0__:keymiss")] string entry, [CosmosDB(
```

Comment:
@mapalan Can you help Nadia get a better understanding of how to use this trigger in this function?